frameworks/helloworld/tests/test_multistep_plan.py

```diff
@@ -40,9 +39,7 @@ def close_enough(val0, val1):
     sdk_tasks.check_tasks_updated(config.SERVICE_NAME, "hello", hello_ids)
     config.check_running(config.SERVICE_NAME)
 
-    all_tasks = shakedown.get_service_tasks(config.SERVICE_NAME)
-    running_tasks = [
-        t for t in all_tasks if t["name"].startswith("hello") and t["state"] == "TASK_RUNNING"
-    ]
+    all_tasks = sdk_tasks.get_service_tasks(config.SERVICE_NAME, task_prefix="hello")
+    running_tasks = [t for t in all_tasks if t.state == "TASK_RUNNING"]
     for t in running_tasks:
-        assert close_enough(t["resources"]["cpus"], updated_cpus)
+        assert close_enough(t.resources["cpus"], updated_cpus)
```

Comment:
> In terms of general usablity, I think having the explicit class is more useful since you can look to the class definition to see what members are available

Like 10000% 🌟 

I later saw the `Task` class. I really like it!